jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -435,8 +435,26 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
       this.state.changeState(storeState);
     };
 
+    const addGetStartedAnnotation = (entityId) => {
```

Comment:
This function is only used internally. Make it receive a url already? So that we don't even have to guess legacy vs new id?

Reply:
I decided not to do that as it would mean adding more complexity to the post-deployment component. Happy to change if you feel strongly about it though.